src/types.ts

```diff
@@ -0,0 +1,11 @@
+export interface ISessionStorage {
+  close(): Promise<void>;
+  set(key: string, value: any, expires_sec: number): Promise<void>;
+  tryGet(key: string): Promise<any | null>;
```

Comment:
Why not just `get`?

Reply:
Also don't really like the `any` here and below. Can we specify actual types?

Reply:
FIXED